OpenRA.Mods.RA/Widgets/Logic/ObserverStatsLogic.cs

```diff
@@ -272,8 +272,11 @@ ScrollItemWidget BasicStats(Player player)
 
 			var stats = player.PlayerActor.TraitOrDefault<PlayerStatistics>();
 			if (stats == null) return template;
-			template.Get<LabelWidget>("KILLS").GetText = () => (stats.UnitsKilled + stats.BuildingsKilled).ToString();
-			template.Get<LabelWidget>("DEATHS").GetText = () => (stats.UnitsDead + stats.BuildingsDead).ToString();
+			var totalKilled = stats.UnitsKilled + stats.BuildingsKilled;
```

Comment:
Pulling these outside the lambda will prevent them from updating while the menu is open.